src/main/kotlin/com/cognifide/gradle/aem/environment/tasks/EnvDown.kt

```diff
@@ -0,0 +1,29 @@
+package com.cognifide.gradle.aem.environment.tasks
+
+import com.cognifide.gradle.aem.environment.ServiceAwait
+import com.cognifide.gradle.aem.environment.docker.DockerTask
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.TaskAction
+
+open class EnvDown : DockerTask() {
+
+    init {
+        description = "Stops local development environment " +
```

Comment:
in case of aem tasks I used `Turn on ` `Turn off` etc

Reply:
OK